employee-rostering-frontend/src/ui/components/WeekPicker.css

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ .week-picker-container {
+    display: "flex";
```

Comment:
Should be `display: flex;` (without quotes).

https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
Or maybe it shouldn't be there at all. The thing is, with quotes it's invalid and behaves like the default `display: block;`. I tried `display: flex;` and it looks weird. It's probably not what you want...

Reply:
Fixed; remove display property